### ISSUE TITLE
LIBRARY_SEARCH_PATHS[sdk=iphonesimulator*][arch=x86_64] doesn't work

### ISSUE THREAD

**Issue by jzxchiang1** [TimNN/cargo-lipo#58]
When I supply `[arch=x86_64]` or `[arch=arm64]` to my LIBRARY_SEARCH_PATHS envvar, it doesn't work and the envvar is not populated correctly. When I change it to `[arch=*]` or remove that condition, it works. `[sdk=...]` doesn't have this problem.

This seems to be a trend:

- https://developer.apple.com/forums/thread/699984
- https://stackoverflow.com/questions/70999932/architecture-specific-build-settings-in-xcode-13

FWIW, I printed out my build settings using `xcodebuild ... -showBuildSettings` and confirmed that the envvar doesn't work with `[arch=x86_64]` or `[arch=arm64]`.

Has anyone seen this before? Thanks!

**Comment by ivanmoskalev** [comment]
Yes, same problem, had to fall back to good ol' trusty `cargo-lipo`
